### DISCUSSION TITLE
Does module federation work in Remix?

### DISCUSSION THREAD

**Discussion by AsuraKev** [module-federation/module-federation-examples#3177] [Q&A]
Quick question here, I noticed that Modern js and Remix are extremely similar. After hearing that Module Federation 1.1 is going to be rolled out to Modern Js, I am wondering will it be available also on the Remix side?

thanks :)

**Comment by alioshr** [comment]
anyone?

**Comment by ScriptedAlchemy** [comment]
Yes, it does. You just need to configure the build.

ModernJS is Remix-based, and it's what we use at ByteDance, who heavily uses federation.

**Reply by alioshr** [reply]
What you are saying is that it lacks a out-of-the-box integration, right? I see that we have it for NextJs and ModernJs.

**Reply by ScriptedAlchemy** [reply]
Yeah lacks OOTB for remix. But we know it works since modernjs is remix, and ByteDance has deployed thousands of them.
